docs/ref/forms/fields.txt

```diff
@@ -1077,14 +1079,24 @@ objects (in the case of ``ModelMultipleChoiceField``) into the
 
 For more complex uses, you can specify ``queryset=None`` when declaring the
 form field and then populate the ``queryset`` in the form's ``__init__()``
-method::
+method:
+
+.. code-block:: python
+
+    class Country(models.Model):
```

Comment:
Not sure what's wrong with the existing example from ced3e303? This seems to add more complexity in trying to understand the essence of the issue (of course, I wrote the original patch, so I'm biased). The example no longer demonstrates `queryset=None` as stated on line 1080.